include/deal.II/fe/fe_interface_values.h

```diff
@@ -51,6 +51,12 @@ template <int dim, int spacedim = dim>
 class FEInterfaceValues
 {
 public:
+  /**
```

Comment:
Do we put the public member variables at the top above the constructor? I would prefer them to be below the functions.

Reply:
I just copied the style from `FEValuesBase` :-)